sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -636,6 +637,8 @@ export interface FileDownloadHeaders {
 
 // @public
 export interface FileDownloadOptionalParams extends coreHttp.OperationOptions {
+    allowTrailingDot?: boolean;
```

Comment:
I don't remember why these two interfaces `FileDownloadOptionalParams` and `FileUploadRangeFromURLOptionalParams` were exported. They are from generated code?

Reply:
ah, it was from the beginning, probably accidentally :(